### PR TITLE
Enabling of STEPDOWN feature on unused IOBs

### PR DESCRIPTION
**Pull request by mkurc-ant**
This PR modifies `fasm2frames` so it detects IOBs which have the STEPDOWN feature enabled and sets the same feature on all other unused IOBs within the same bank.

There is also added dumping of IO bank numbers along with part pinout (`*_package_pins.csv` files)

### CODE REVIEW

File: utils/fasm2frames.py [hunk 1]
```diff
@@ -142,6 +175,41 @@ def check_for_pudc_b(set_feature):
         if missing_features:
             raise fasm_assembler.FasmLookupError('\n'.join(missing_features))
 
+    if part is not None:
+        # Make a set of all used IOB tiles and sites. Look for the "STEPDOWN"
```
Comment (litghost) [inline utils/fasm2frames.py]: Make sure there is not conflict (e.g. an IOB that requires the STEPDOWN, and an IOB that does NOT have STEPDOWN).

**Comment by litghost** [comment]
CI is red?

File: utils/fasm2frames.py (hunk 1 above)
Comment (mkurc-ant) [reply utils/fasm2frames.py]: Are there any IOBs that do not have STEPDOWN? How do I identify them?

**Comment by mkurc-ant** [comment]
@litghost The CI is red because I added the bank number to `*_package_pins.csv`. Am I correct?

File: utils/fasm2frames.py (hunk 1 above)
Comment (litghost) [reply utils/fasm2frames.py]: Sorry, I meant IOB's that have a IOSTANDARD that should not have STEPDOWN set, e.g. LVCMOS33.

**Comment by litghost** [comment]
```
+ python3 /tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/utils/fasm2frames.py design.fasm design.frm
Traceback (most recent call last):
  File "/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/utils/fasm2frames.py", line 287, in <module>
    main()
  File "/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/utils/fasm2frames.py", line 283, in main
    emit_pudc_b_pullup=args.emit_pudc_b_pullup)
  File "/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/utils/fasm2frames.py", line 192, in run
    tile, site, tag = feature.split(".", maxsplit=2)
ValueError: not enough values to unpack (expected 3, got 2)
Makefile:14: recipe for target 'build/design.bit' failed
make[2]: *** [build/design.bit] Error 1
make[2]: Leaving directory '/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/minitests/roi_harness'
Makefile:24: recipe for target 'run' failed
make[1]: *** [run] Error 2
make[1]: Leaving directory '/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/minitests/roi_harness'
Makefile:143: recipe for target 'db-extras-artix7' failed
make: *** [db-extras-artix7] Error 2
```

**Comment by litghost** [comment]
The bug is here: `tile, site, tag = feature.split(".", maxsplit=2)`

You assumed that all features have at least 3 parts, that isn't always true.

```+ python3 /tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/utils/fasm2frames.py design.fasm design.frm
Traceback (most recent call last):
  File "/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/utils/fasm2frames.py", line 287, in <module>
    main()
  File "/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/utils/fasm2frames.py", line 283, in main
    emit_pudc_b_pullup=args.emit_pudc_b_pullup)
  File "/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/utils/fasm2frames.py", line 192, in run
    tile, site, tag = feature.split(".", maxsplit=2)
ValueError: not enough values to unpack (expected 3, got 2)
Makefile:14: recipe for target 'build/design.bit' failed
make[2]: *** [build/design.bit] Error 1
make[2]: Leaving directory '/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/minitests/roi_harness'
Makefile:24: recipe for target 'run' failed
make[1]: *** [run] Error 2
make[1]: Leaving directory '/tmpfs/src/github/symbiflow-prjxray-presubmit-db-artix7/minitests/roi_harness'
Makefile:143: recipe for target 'db-extras-artix7' failed
make: *** [db-extras-artix7] Error 2
```

**Comment by mkurc-ant** [comment]
Right, fixed that.

**Comment by litghost** [comment]
Need to be rebased/merged with master

**Pull request merged**